roles/acme/tasks/challenge/dns-01/autodns.yml

```diff
@@ -9,7 +9,7 @@
         method: POST
         body_format: json
         body: { context: "4", user: "{{ acme_dns_user }}", password: "{{ acme_dns_password }}" }
-      register: login
+      register: acme_login_result
```

Comment:
is it `acme_login` (like below) or `acme_login_result`?

Reply:
fixed